kernel/src/main/java/org/kframework/utils/inject/JCommanderModule.java

```diff
@@ -74,6 +74,6 @@ public void println(String msg) {
     String usage(JCommander jc) {
         StringBuilder sb = new StringBuilder();
         usage(jc, sb);
-        return StringUtil.finesseJCommanderUsage(sb.toString(), jc)[0];
+        return sb.toString().replaceAll("        ", "    ");
```

Comment:
What is happening here?

Reply:
Ah, I see:
> //for some reason the usage pattern indents commands inconsistently, so we need to adjust it

Please bring this comment here.

Reply:
Done.